### ISSUE TITLE
Indent rule with VariableDeclarator set to "first"

### ISSUE THREAD

**Issue by bmblb** [eslint/eslint#11265]
**Tell us about your environment**

* **ESLint Version:**
5.12.0
* **Node Version:**
8.13.0
* **npm Version:**
6.4.1

**What parser (default, Babel-ESLint, etc.) are you using?**
default

Hello. I am trying **VariableDeclarator: first** option for **indent** rule and it breaks alignment. 
This is the example of *correct* code:
```js
let a = {
        a : 1,
        b : 2
    },
    b = 2,
    c = 3;

const d = {
        a : 1,
        b : 2
    },
    e = 3;
```

When I add rule `indent: ["error", 4, { "VariableDeclarator": "first" }]`, eslint consider code above invalid and changes it to below:
```js
/* eslint indent: ["error", 4, { "VariableDeclarator": "first" }] */
let a = {
    a : 1,
    b : 2
},
    b = 2,
    c = 3;

const d = {
    a : 1,
    b : 2
},
      e = 3;
```

**Comment by bmblb** [comment]
Related PR: https://github.com/eslint/eslint/pull/11193
@g-plane could you take a look?

**Comment by g-plane** [comment]
IMHO, it's valid. Here is an example which the option isn't `first` but behaves similarly: https://eslint.org/demo/#eyJ0ZXh0IjoiLyogZXNsaW50IGluZGVudDogW1wiZXJyb3JcIiwgMiwgeyBcIlZhcmlhYmxlRGVjbGFyYXRvclwiOiAwIH1dICovXHJcbmxldCBhID0ge1xyXG4gIGEgOiAxLFxyXG4gIGIgOiAyXHJcbn0sXHJcbmIgPSAyLFxyXG5jID0gM1xyXG4iLCJvcHRpb25zIjp7InBhcnNlck9wdGlvbnMiOnsiZWNtYVZlcnNpb24iOjksInNvdXJjZVR5cGUiOiJzY3JpcHQiLCJlY21hRmVhdHVyZXMiOnt9fSwicnVsZXMiOnt9LCJlbnYiOnt9fX0=

I think you may need to configure the option `ObjectExpression`: https://eslint.org/docs/rules/indent#objectexpression
